scripts/migrate_to_gencc.pl

```diff
@@ -0,0 +1,201 @@
+=head1 LICENSE
+ 
+See the NOTICE file distributed with this work for additional information
+regarding copyright ownership.
+ 
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+ 
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+ 
+=cut
+use strict;
+use warnings;
+
+use Bio::EnsEMBL::Registry;
+use Getopt::Long;
+
+use FileHandle;
+
+my $config = {};
+GetOptions(
+  $config,
+  'registry_file=s',
+) or die "Error: Failed to parse command line arguments\n";
+
+die ('A registry file is required (--registry_file)') unless (defined($config->{registry_file}));
+
+# Instatiate registry object 
+my $registry = 'Bio::EnsEMBL::Registry';
+my $registry_file = $config->{registry_file};
+$registry->load_all($registry_file);
+my $species = 'human';
+
+my $mappings = {
+  'original_allelic_requirement' => {
+    'allelic_requirement' => {
+      'biallelic' => 'biallelic_autosomal',
+      'hemizygous' => 'monoallelic_X_hem',
+      'mitochondrial' => 'mitochondrial',
+      'digenic' => 'mitochondrial',
```

Comment:
These seem like strange mappings:
- 'digenic' => 'mitochondrial'
- 'uncertain' => 'mitochondrial'
is this correct? Do they arise because there are only a couple of examples of each which happen to be mitochondrial?

Reply:
No it is not. I did that because we are still waiting for feedback for some of them from the curators (e.g imprinted, digenic).  Uncertain will be dropped and mosaic will be mapped to cross cutting modifier. So i am using that mapping as a placeholder for now.

Reply:
Thanks! So this is not to be merged until we have that information. We could consider a different placeholder if it takes a while to get the info